content/admin/Advanced Configuration Options.adoc

```diff
@@ -0,0 +1,170 @@
+
+For the parameter `dirpass:[_]mode` you can set the value to 1528,
+which is the decimal equivalent of the octal value 02770. For the parameter
+`filepass:[_]mode` you can set the value to 432, which is the decimal equivalent of the octal value 0660.
+
+== Changing the location of the download folder
+
+All files downloaded by the system are stored in a special download folder, its location is set by variable
+`uploadpass:[_]dir` in file `config.php`. By default, this is the upload folder, located in the installed system folder.
+If necessary, its location can be changed, for this purpose in a variable `uploadpass:[_]dir` set absolute
+The path to the new location of the download folder.
+
+{{% notice note %}}
+Do not forget to copy all the files from the old download folder to the new one.
+{{% /notice %}}
+
+The new folder must be writable by the user who is running the web server.
+
+== Changing the location of the cache folder
+
+The system actively uses data caching, which greatly improves its performance.
+A special folder is used to store cached data (compiled templates, email data, etc.)
+and its size can be quite large. By default, this is the cache folder located in the installed system folder.
+If necessary, its location can be changed, for this purpose in a variable
+`cachepass:[_]dir`, located in the file `config.php`, set the absolute path to the new
+location of the cache folder. At the same time, the link pointing to the new cache
+folder should be placed in the same place of the cache folder.
+This is necessary to provide access to some cached system files.
+The new folder must be writable by the user who is running the web server.
+
+== Permanent display of Advanced Filters in List Forms
+
+In SuiteCRM version 7.8, a more compact was introduced instead of Basic and Advanced Search
+functional link:../../user/introduction/user-interface/search/[of filters].
+If you still need a permanent display of filter fields at the top of the List Form
+certain modules (as it was in earlier versions of the system), add the appropriate
+array by analogy with the following example (for modules *Accounts*, *Users* and *Employees*)
+to file `config.php`:
+
+[source,php]
+'enable_legacy_search' =>
+  array (
+    0 => 'Accounts',
+    1 => 'Users',
+    2 => 'Employees',  ),
+
+either the corresponding lines in the file `config_override.php`, example :
+
+[source,php]
+$sugar_config['enable_legacy_search'][] = 'Accounts';
+$sugar_config['enable_legacy_search'][] = 'Users';
+$sugar_config['enable_legacy_search'][] = 'Employees';
+
+
+== Subpanels: replacing the drop-down menu with buttons
+
+If necessary, you can display the functions of a particular
+sub-panel not in the form of drop-down menu items, but in the form of buttons.
+The figure below shows an example of a standard sub-panel
+*Measures* and a modified sub-panel *history* in the module *Pre-contacts*:
+
+image:measures-subpanel.png[Replacing the drop-down menu with buttons]
```

Comment:
I suggest using this image instead:

![image](https://user-images.githubusercontent.com/15945027/52855071-4b12a500-3118-11e9-807d-1911f7ef7f18.png)